node/utils/simulation.ts

```diff
@@ -161,6 +163,12 @@ const getDiscountHighLights = (ratesAndBenefitsData: RatesAndBenefitsData) => {
     .map((rateAndBenefitsIdentifier: any) => ({
       '<Name>k__BackingField': rateAndBenefitsIdentifier.name,
       ...rateAndBenefitsIdentifier,
+      additionalInfo: Object.keys(rateAndBenefitsIdentifier.additionalInfo).map(
```

Comment:
Can `rateAndBenefitsIndentifier.additionalInfo` arrive as null or undefined? In that case `Object.keys` would throw an error.

Reply:
Done! And yes it would crash to null values, thanks for the heads up.